### PR TITLE
Feature/timetable creation

### PR DESCRIPTION
**Pull request by mbouilloud**

### CODE REVIEW

**Comment by mbouilloud** [comment]
I will add the feature, that a click on a course/room in the timetable opens the review activity, in my next commit.

**Comment by mbouilloud** [comment]
In my next commit, I will do a package refactor, some comment additions and view model cleaning for profile.

File: build.gradle [hunk 1]
```diff
@@ -24,6 +24,7 @@ allprojects {
     repositories {
         google()
         mavenCentral()
+        jcenter()
```
[the remark this answers is not in the document]
Comment (mbouilloud) [reply build.gradle:27]: The day selector part of the UI comes from this repo, it's not in the others.

**Review by jmettler27 — APPROVED** [review]
Cool feature

**Review by NicolasMatekalo — APPROVED** [review]
The duplications are hard too fix but I don't think it is a big deal. 
Anyways,, great job!

**Pull request merged**